Task 5/script.js

```diff
@@ -19,3 +19,23 @@ const users = [
   { id: '8', name: 'Simon Peterson', age: 30, hasDog: false },
   { id: '9', name: 'Daniel Cane', age: 51, hasDog: true },
 ];
+
+const filterDogOwers = ((users) => {
```

Comment:
Little bit too many brackets, no need `(` in the beginning and no need for this `)` in the end.
Also function should return new array not just log every object.